sql-plugin/src/main/scala/com/nvidia/spark/rapids/RegexParser.scala

```diff
@@ -675,6 +675,28 @@ class CudfRegexTranspiler(mode: RegexMode) {
             RegexChar(' '), RegexChar('\u000b'))
           chars ++= Seq('n', 't', 'r', 'f').map(RegexEscaped)
           RegexCharacterClass(negated = ch.isUpper, characters = chars)
+        case 'h' | 'H' =>
```

Comment:
nit: Where did you get these unicode character lists from?  Might be nice to include a comment so others can reproduce the work or update it if needed.